index.html

```diff
@@ -163,6 +179,25 @@ <h3>Multi-Post Stories</h3>
   </section>
   <!-- The Contact Us Form -->
   <section>
+    <div class="desktop">
+      <div class="form-content">
+        <h2>Let’s chat!</h2>
+        <p>
+          If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+        </p>
+        <form class="form-container" action="https://formspree.io/f/xdobrbdn" method="post">
+        <div class="row">
+          <input id="name" name="Name" class="input-name" type="text" placeholder="Your Name" maxlength="30" required>
+          <input id="name" name="Name" class="input-name" type="text" placeholder="Last Name" maxlength="30" required>
+        </div>
+          <input id="email" name="userEmail" class="input-email" type="email" placeholder="Email Address" required>
+          <textarea name="Message" id="text-message" rows="10" maxlength="500" placeholder="Write your message here"
+            required></textarea>
+          <button type="submit" class="button" id="button">See Project</button>
+        </form>
+      </div>
+    </div>
+    <span>
     <div class="form-content">
       <h2>CONTACT US</h2>
       <p>
```

Comment:
kindly centralize the contact form on the page. This is because it looks better and follows your chosen design.

Your design:
![image](https://user-images.githubusercontent.com/20554583/186683415-72f63a3b-9421-4016-b234-83f5110cd563.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/186683465-f769c82b-44ee-48f3-a520-be255deb2a91.png)